subprojects/ide/src/main/java/org/gradle/plugins/ide/api/GeneratorTask.java

```diff
@@ -116,7 +118,9 @@ public File getInputFile() {
     }
 
     // Workaround for when the task is given an input file that doesn't exist
-    @Nullable  @Optional @InputFile
+    @Nullable  @Optional
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
`IGNORE`?

Reply:
I'll change it to `IGNORE`.

The comment above is also interesting: 
```
     * The input file to load the initial configuration from. Defaults to the output file. If the specified input file
     * does not exist, this task uses some default initial configuration.
```

Do we need to do about the tasks where the input file is the output file?

Reply:
Gah. Do we detect these as stale outputs during the next run?

Reply:
I think the first run for a new version would delete the files, right? Since it would encounter the output file (which is also the input file), has no record of it and then clean it up.

Reply:
Let's just leave this alone for now...